packages/core/src/components/ColorPalette/VariantRow.tsx

```diff
@@ -152,7 +176,7 @@ export function VariantRow({
         css={{
           height: '3rem',
           width: '100%',
-          backgroundColor: `${token.value}`,
+          backgroundColor: `${changing ? color : variant.token.value}`,
```

Comment:
Let's also remove the concept of the changing var @karanvirsb

Reply:
I cannot remove this otherwise the change will only display after the user has stopped changing.